components/NewsletterHome.vue

```diff
@@ -41,17 +40,20 @@ const hideComp = () => {
 
 // submit the newsletter form and add email address to the Gothamist Newsletter list
 const submitForm = (email) => {
+  console.log('submitting')
   isSubmitting.value = true
   submissionStatus.value = null
-  axios
-    .post(config.NEWSLETTER_API, {
+  fetch(config.NEWSLETTER_API, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'Access-Control-Allow-Origin': '*',
+    },
+    body: JSON.stringify({
       list: config.NEWSLETTER_LIST_ID,
       email: email,
-      headers: {
-        'Content-Type': 'application/json',
-        'Access-Control-Allow-Origin': '*',
-      },
-    })
+    }),
+  })
```

Comment:
I think if you use nuxt's $fetch you can shorten this to just something like
```  $fetch(config.NEWSLETTER_API, {
      method: 'POST',
      body: {
        list: config.NEWSLETTER_LIST_ID,
        email: email,
      }
  })
```
https://v3.nuxtjs.org/api/utils/$fetch
https://github.com/unjs/ohmyfetch#%EF%B8%8F-json-body

Reply:
if I remove the "JSON.stringify()" I get this error in VSCODE
![image](https://user-images.githubusercontent.com/13803343/181076415-dbdba60c-d003-4e50-9f08-9fe304f58b70.png)

Reply:
but it seems to submit fine....

Reply:
ah... ok added the "$" to the fetch